packages/grid/x-data-grid/src/models/events/gridEvents.ts

```diff
@@ -259,7 +259,16 @@ enum GridEvents {
    * Fired when the visible rows are updated
    * @ignore - do not document.
    */
-  visibleRowsSet = 'visibleRowsSet',
+  /**
+   * Fired when the sorted rows are updated
+   * @ignore - do not document
+   */
+  sortedRowsSet = 'sortedRowsSet',
+  /**
+   * Fired when the filtered rows are updated
+   * @ignore - do not document.
+   */
+  filteredRowsSet = 'filteredRowsSet',
```

Comment:
This event is never used internally
I created it to handle #1106 but I can remove it from this PR